mayastor/src/bdev/nexus/nexus_channel.rs

```diff
@@ -45,6 +45,8 @@ pub enum DREvent {
     ChildRemove,
     /// Child rebuild event
     ChildRebuild,
+    /// Child status information is being applied
+    ChildApplyStatus,
```

Comment:
Not sure about the name but also don't have good suggestions... maybe something like NexusStartup?

Reply:
I've left it as it is for now. I couldn't think of a better name.

Reply:
ChildStatusSync maybe?

Reply:
Good suggestion, done.